prestoadmin/util/presto_config.py

```diff
@@ -48,42 +47,57 @@ class PrestoConfig:
         HTTPS_PORT_KEY: '8443',
         CLIENT_KEYSTORE_PATH_KEY: None,
         CLIENT_KEYSTORE_PASSWORD_KEY: None,
-        LDAP_CLIENT_USER_KEY: None,
-        LDAP_CLIENT_PASSWORD_KEY: None
+        INTERNAL_COMMUNICATION_SECRET: None
     }
 
-    def __init__(self, config_properties, config_path, config_host):
+    default_node_config = {
+        NODE_ENVIRONMENT: 'presto'
+    }
+
+    def __init__(self, config_properties, node_config_properties, config_path, config_host):
```

Comment:
One question: are the locations of `node.properties` and `config.properties` not configurable? Whenever we need to get their path we do a `os.path.join(REMOTE_CONF_DIR, CONFIG_PROPERTIES)`. It seems like they're not configurable from the usage I've found so far.

If that's the case I think we can get rid of the `config_path` variable here and create two static methods in this class.
```python
@staticmethod
def get_config_path():
    os.path.join(REMOTE_CONF_DIR, CONFIG_PROPERTIES)

@staticmethod
def get_node_config_path():
    os.path.join(REMOTE_CONF_DIR, NODE_PROPERTIES)
```

This way we simplify the constructor here and build these paths in one place instead of inlining them in three places.